test/com/facebook/buck/apple/XctoolRunTestsStepTest.java

```diff
@@ -620,7 +621,7 @@ public void testDirectoryAndLevelPassedInEnvironment() throws Exception {
         Optional.of("TEST_LOG_LEVEL"),
         Optional.of("verbose"),
         Optional.empty(),
-        Optional.of(Paths.get("/path/to/snapshotimages")));
+        Optional.of(Either.ofLeft(Paths.get("/path/to/snapshotimages"))));
```

Comment:
Need to change this, since you made `XctoolRunTestsStep` take an `Optional<String>`.